source/CardGames.py

```diff
@@ -111,10 +112,34 @@ def showHand(self, printShort: bool = False):
           image = card.image[idx] if self.knownCards[i] else card.cardBack[idx]
           print(image, end="")
       print()
+  def High(self):
+    l =[]
+    l2 =[]
+    for c in self.hand:
+      l.append(str(c))
+    for c in l:
+      l2.append(int(c))
+    x = max(l2)
+    print(x)
 
   def clearHand(self):
     self.hand = []
     self.knownCards = []
+  def printCardsIfSet(self):
+    # Check for a set of four cards with the same value
+    value_counts = {}
+    for card in self.hand:
```

Comment:
I like how concise this code is. You fit everything into one for loop and it gets the job done. It's also easy to understand, so good job.

Reply:
Thanks, Keren.